### ISSUE TITLE
Routine update of dependencies

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2897]
Update Poetry managed dependencies within existing pyproject.toml constraints.

```
cd /opt/rockstor
poetry update
```

Issue resolution via resulting updated poetry.lock file.

---

Upstream reference: https://python-poetry.org/docs/basic-usage/#updating-dependencies-to-their-latest-versions

**Comment by phillxnet** [comment]
# Resulting Poetry report
Leap 15.6 X86_64 host

```
Package operations: 0 installs, 14 updates, 0 removals

  • Updating cffi (1.16.0 -> 1.17.0)
  • Updating cryptography (42.0.8 -> 43.0.0)
  • Updating more-itertools (10.3.0 -> 10.4.0)
  • Updating zipp (3.19.2 -> 3.20.1)
  • Updating idna (3.7 -> 3.8)
  • Updating importlib-metadata (8.0.0 -> 8.4.0)
  • Updating jaraco-context (5.3.0 -> 6.0.1)
  • Updating jaraco-functools (4.0.1 -> 4.0.2)
  • Updating setuptools (70.3.0 -> 73.0.1)
  • Updating django (4.2.14 -> 4.2.15)
  • Updating keyring (25.2.1 -> 25.3.0)
  • Updating zope-interface (6.4.post2 -> 7.0.2)
  • Updating gunicorn (22.0.0 -> 23.0.0)
  • Updating pyzmq (26.0.3 -> 26.2.0)

Writing lock file
```

**Comment by phillxnet** [comment]
# Notable Changelog references:

- django: https://docs.djangoproject.com/en/5.0/releases/4.2.15/
- keyring: https://github.com/jaraco/keyring/blob/main/NEWS.rst
- gunicorn: https://docs.gunicorn.org/en/stable/news.html
- pyzmq: https://pyzmq.readthedocs.io/en/latest/changelog.html

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2898
